file/lib/file_source_endpoint.ex

```diff
@@ -182,8 +195,11 @@ defmodule Membrane.RTC.Engine.Endpoint.File do
 
   @impl true
   def handle_parent_notification(:start, _ctx, state) do
-    track_ready = {:track_ready, state.track.id, :high, state.track.encoding}
-    {[notify_parent: track_ready], state}
+    {get_track_ready_notification(state), state}
+  end
```

Comment:
I think we should handle `:start` notification based on `:autoplay` value.
If the `:autoplay` value is true we should log some warning.

Also what will happen if we send `:track_ready` action twice?

Reply:
If we try to send `track_ready` notification twice we get error from Membrane - trying to link the input pads twice.